### PR TITLE
[INLONG-8749][Manager] Support configuring whether to initialize node state during startup

### LINKED ISSUE #8749: [Improve][Manager] Support configuring whether to initialize node state during startup
### Description

Support configuring whether to initialize node state during startup.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8749 

### Motivation

Support configuring whether to initialize node state during startup.

### Modifications

Support configuring whether to initialize node state during startup.

### CODE REVIEW

File: inlong-manager/manager-web/src/main/resources/application-prod.properties [hunk 1]
```diff
@@ -87,6 +87,7 @@ sort.enable.zookeeper=false
 
 # cluster node timeout interval of heartbeat unit: second, the interval multiplied by 5 represents the true heartbeat timeout interval
 cluster.heartbeat.interval=6
+reset.nodeStatus.enabled=false
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/resources/application-prod.properties:90]: Please add an annotation for the new param.

File: inlong-manager/manager-web/src/main/resources/application-test.properties [hunk 2]
```diff
@@ -89,6 +89,9 @@ sort.enable.zookeeper=false
 # cluster node timeout interval of heartbeat unit: second, the interval multiplied by 5 represents the true heartbeat timeout interval
 cluster.heartbeat.interval=6
 
+#
+reset.nodeStatus.enabled=false
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/resources/application-test.properties:93]: Ditto, otherwise, please remove the empty line.

Comment (fuweng11) [reply inlong-manager/manager-web/src/main/resources/application-test.properties:91]: Fix.

File: inlong-manager/manager-web/src/main/resources/application-prod.properties (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-web/src/main/resources/application-prod.properties:92]: Done.

**Review by luchunliang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**